app/main.py

```
@@ -595,6 +594,33 @@ async def chat_image(
+        return {"error": "Login Required"}
+    user_id = user_id_from_jwt(user_jwt)
+    if not is_uuid_like(user_id):
+        return {"error": "Login Required"}
+    if not is_uuid_like(image_id):
+        return {"error": "Image invalid"}
+    sql_connection.ping()
+    sql_connection.cursor().execute(
+        f"CREATE TABLE IF NOT EXISTS `{mariadb_name(user_id, chat_id)}`.`images` (id char(36) NOT NULL,content TEXT,PRIMARY KEY(id)) charset=utf8;")
+    sql_connection.cursor().execute(f"ALTER TABLE `{mariadb_name(user_id, chat_id)}`.`images` ADD COLUMN IF NOT EXISTS alt TEXT;")
+    cursor = sql_connection.cursor()
+    cursor.execute(f"SELECT content, alt FROM `{mariadb_name(user_id, chat_id)}`.`images` WHERE id=?", (image_id,))
```

Ruff (0.13.1)

Possible SQL injection vector through string-based query construction

(S608)